README.md

```diff
@@ -1,2 +1,50 @@
-# pets-exotica
- An e-commerce app for pets essentials. 
+# [PETS EXOTICA](https://pets-exotica.vercel.app)
+
+One stop online shop for all your pet needs.
+## **PREVIEW**
+
+<br/>
+
+![MEOW-ME Preview GIF](./assets/pets-exotica.gif)
+
+<br/>
+
+## **Table of contents**
+
+- [Preview](#preview)
+- [Feature](#documentation)
+- [Technologies used](#technologies-used)
+- [Author](#author)
+
+
+<br/>
+
+## **Features**
+
+
+Browse the **[website](https://pets-exotica.vercel.app)** here.
+- [Home page](https://pets-exotica.vercel.app)
+- [Product listing page & Filters](https://pets-exotica.vercel.app/product-listing/products.html)
+- [Cart Management](https://pets-exotica.vercel.app/cart/cart.html)
+- [Wishlist Management](https://pets-exotica.vercel.app/wishlist/wishlist.html)
+- [Signup](https://pets-exotica.vercel.app/authentication/signup/signup.html)
+- [Login](https://pets-exotica.vercel.app/authentication/login/login.html)
+- [Logout](https://pets-exotica.vercel.app/wishlist/wishlist.html) (Logout Button on wishlist page)
+- [Forget Password](https://pets-exotica.vercel.app/authentication/forget-password/forget-password.html)
+- [Remember me while login](https://pets-exotica.vercel.app/authentication/login/login.html)
+
+<br/>
+
+## **Technologies used**
+
+- HTML5
+- CSS3
+
+<br/>
+
+## **Author**
+<hr>
```

Comment:
You can remove this as titles come with line  below them
![image](https://user-images.githubusercontent.com/77927918/155193349-4154597f-53b3-4790-bb72-ec15a7b3503e.png)